TombEngine/Game/misc.cpp

```diff
@@ -59,15 +59,22 @@ bool IsNextSectorValid(const ItemInfo& item, const Vector3& dir, float dist)
 	{
 		// Test for step.
 		int relFloorHeight = abs(pointColl.Position.Floor - item.Pose.Position.y);
-		if (relFloorHeight >= CLICK(1) && item.Pose.Position.y >= pointColl.Position.Floor)
-			return false;
+
+		if (relFloorHeight >= CLICK(1) && item.Pose.Position.y >= pointColl.Position.Floor && canFloat)
```

Comment:
Remove double tabs on return lines. Block like this one can be formated as follows:

if (x)
{
    // do A
}
else if (y)
{
    // do B
}

Reply:
fixed